### DISCUSSION TITLE
ZRangeByScores是否支持开区间

### DISCUSSION THREAD

**Discussion by dayansoft01** [2881099/FreeRedis#58] [General]
翻阅源码发现好像是不支持开区间，作者有这方面的考虑吗

**Comment by 2881099** [comment]
https://github.com/2881099/FreeRedis/blob/9b29e4a22c8954e58f6486d850fc86ac692b69c0/src/FreeRedis/RedisClient/SortedSets.cs#L79

min, max 吗？有这两个参数。

**Reply by dayansoft01** [reply]
默认情况下，区间的取值使用闭区间 (小于等于或大于等于)，你也可以通过给参数前增加 ( 符号来使用可选的开区间 (小于或大于)。

`ZRANGEBYSCORE zset (1 5`
返回所有符合条件 1 < score <= 5 的成员

`ZRANGEBYSCORE zset (5 (10`
则返回所有符合条件 5 < score < 10 的成员

这样的开区间命令我们有没有进行支持，我从源码上看好像没有支持。

**Comment by 2881099** [comment]
https://github.com/2881099/FreeRedis/blob/9b29e4a22c8954e58f6486d850fc86ac692b69c0/src/FreeRedis/RedisClient/SortedSets.cs#L212

看重载方法，有 string min, string max

**Reply by dayansoft01** [reply]
嗯，明白了，能否像增加一个枚举类型呢，就像是StackExchange.Redis一样，感觉这样更易用一些。

**Reply by 2881099** [reply]
> 嗯，明白了，能否像增加一个枚举类型呢，就像是StackExchange.Redis一样，感觉这样更易用一些。

这个不一定，常看 redis.doc 就明白这个了，没有二次理解成本。
